Model/Source/AbstractTable.php

```diff
@@ -26,11 +28,13 @@ public function __construct(
         ProductHelper $producthelper,
         CategoryHelper $categoryHelper,
         ConfigHelper $configHelper,
+        ProxyHelper $proxyHelper,
```

Comment:
Seems that `proxyHelper` is a dependency of the `Facets` class only. I would override the `__construct` of the `Facets` instead. What do you think?

Reply:
That's totally true !